lib/validation.ts

```
@@ -0,0 +1,96 @@
+				schema[field.id] = emailSchema;
+				break;
+			case "number":
+				const numberSchema = z.preprocess(
+					(value) => {
+						if (value === "") return undefined;
+						return value;
+					},
+					field.required
+						? z.coerce.number({ error: "This field is required" })
+						: z.coerce.number().optional()
+				);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)